spa/src/components/errors/ChunkErrorBoundary.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import * as Sentry from '@sentry/react';
+
+import * as S from '../../elements/buttons/Button.styled';
+import * as ErrorS from './ErrorFallback.styled';
+
+const ChunkErrorFallback = (error) => {
+  return (
+    <>
+      <ErrorS.ErrorWrapper>
+        <ErrorS.ErrorHeading layout>
+          <h2>😰 We've encountered a problem!</h2>
+          <h4>Click below to reload the page</h4>
+        </ErrorS.ErrorHeading>
+        <S.Button
+          onClick={() => {
+            window.location.reload();
+          }}
+        >
+          Refresh
+        </S.Button>
+      </ErrorS.ErrorWrapper>
+    </>
+  );
+};
+
+const ChunkErrorBoundary = (props) => {
+  return (
+    <Sentry.ErrorBoundary fallback={({ error }) => ChunkErrorFallback(error)}>{props.children}</Sentry.ErrorBoundary>
```

Comment:
I think we can jsut do this:

```suggestion
    <Sentry.ErrorBoundary fallback={<ChunkErrorFallback />}>{props.children}</Sentry.ErrorBoundary>
```

Or something like it.